### PR TITLE
Bugfix MTE-1560 Fix Firebase Physical Device Workflow

### PR DESCRIPTION
**Pull request by jjSDET**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-1560)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/TODO)

## :bulb: Description
We need to update the gcloud command for devices being tested.

The change being made is to simply add `device` to the gcloud argument.

Also removing optional `xcode-version` argument and updating iOS `version` to `16.6`

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [x] Wrote unit tests and/or ensured the tests suite is passing
- [x] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [x] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -1542,8 +1542,9 @@ workflows:
             $HOME/google-cloud-sdk/bin/gcloud version
             $HOME/google-cloud-sdk/bin/gcloud auth activate-service-account -q --key-file /tmp/key-file.json
             $HOME/google-cloud-sdk/bin/gcloud config set project "$GOOGLE_CLOUD_PROJECT"
-            $HOME/google-cloud-sdk/bin/gcloud firebase test ios run --test /Users/vagrant/git/Fennec_Enterprise.zip \
-                --model=iphone14pro,version=16.4 \
+            $HOME/google-cloud-sdk/bin/gcloud firebase test ios run \
+                --test /Users/vagrant/git/Fennec_Enterprise.zip \
+                --device model=iphone14pro,version=16.4 \
                 --xcode-version=14.3.1
```
Comment (isabelrios) [inline bitrise.yml:1548]: I haven't seen xcode 14.3.1 on Firebase, just 14.3

File: bitrise.yml [hunk 2]
```diff
@@ -1542,8 +1542,9 @@ workflows:
             $HOME/google-cloud-sdk/bin/gcloud version
             $HOME/google-cloud-sdk/bin/gcloud auth activate-service-account -q --key-file /tmp/key-file.json
             $HOME/google-cloud-sdk/bin/gcloud config set project "$GOOGLE_CLOUD_PROJECT"
-            $HOME/google-cloud-sdk/bin/gcloud firebase test ios run --test /Users/vagrant/git/Fennec_Enterprise.zip \
-                --model=iphone14pro,version=16.4 \
+            $HOME/google-cloud-sdk/bin/gcloud firebase test ios run \
+                --test /Users/vagrant/git/Fennec_Enterprise.zip \
+                --device model=iphone14pro,version=16.4 \
```
Comment (isabelrios) [inline bitrise.yml:1547]: I don't think this version is available on Firebase...
Screenshot 2023-09-19 at 18 32 33

Comment (jjSDET) [reply bitrise.yml]: oh god, you're right. just updated it. good catch!

File: bitrise.yml (hunk 1 above)
Comment (jjSDET) [reply bitrise.yml:1548]: it worked during my testing, but i won't risk it. mightve just been my sandbox. updating now.

Comment (jjSDET) [reply bitrise.yml]: actually, i'm removing it since it's optional. don't know why i included; latest should always be used by default.

**Review by AaronMT — APPROVED** [review]

**Comment by AaronMT** [comment]
```
% gcloud firebase test ios list-device-capacities
┌─────────────┬───────────────────────┬───────────────┬─────────────────┐
│   MODEL_ID  │       MODEL_NAME      │ OS_VERSION_ID │ DEVICE_CAPACITY │
├─────────────┼───────────────────────┼───────────────┼─────────────────┤
│ ipad5       │ iPad (5th generation) │ 15.4          │ Low             │
│ ipadmini4   │ iPad mini 4           │ 15.4          │ Low             │
│ iphone11pro │ iPhone 11 Pro         │ 14.7          │ Medium          │
│ iphone11pro │ iPhone 11 Pro         │ 16.5          │ Medium          │
│ iphone11pro │ iPhone 11 Pro         │ 16.6          │ High            │
│ iphone12pro │ iPhone 12 Pro         │ 14.8          │ Medium          │
│ iphone13pro │ iPhone 13 Pro         │ 15.2          │ Low             │
│ iphone13pro │ iPhone 13 Pro         │ 15.7          │ Medium          │
│ iphone14pro │ iPhone 14 Pro         │ 16.5          │ High            │
│ iphone14pro │ iPhone 14 Pro         │ 16.6          │ High            │
│ iphone8     │ iPhone 8              │ 14.7          │ Medium          │
│ iphone8     │ iPhone 8              │ 15.7          │ Medium          │
│ iphone8     │ iPhone 8              │ 16.5          │ Medium          │
│ iphone8     │ iPhone 8              │ 16.6          │ High            │
└─────────────┴───────────────────────┴───────────────┴─────────────────┘
```

iPhone14Pro 16.6 High Capacity, LGTM

**Pull request merged**